ui/src/views/PageNotFoundView.vue

```diff
@@ -6,26 +6,19 @@
   >
     <v-row justify="center">
       <v-col cols="auto">
-        page not found
-        <!-- to do: check the log in state, redirect based on the user type -->
-        <v-btn
-          depressed
-          large
-          to="/"
-        >
-          back to home
-        </v-btn>
+        <content-card 
+          title="Page not found" 
+          description="The page either doesn't exist or you don't have access to this page" 
+          :image="require('@assets/svg/404.png')"
+        />
```

Comment:
This is an example of self closing tag. This kind of tag is used when the component doesnt have children; but to be able to use slot, we cannot use this approach.

Instead, we can do it like this:
```

        <content-card>
        Lorem ipsum dolor <router-link> your link </router-link>
        </content-card>
```

This way, the lorem ipsum tag will be printed under the back to home button, since I put the slot under that button (if i remembered it well, please check)

Reply:
Ah sorry, I confused this page with the SignUpThankYouView page. This one is OK already, thank you. You can use that approach with slot for the SignUpThankYouView page.

Reply:
Also that email XD

Reply:
Yes, I learned about the self closing tag. The 404 page does not need children (i.e. link under the back button) so I don't use it there. Screenshots on the PR have been updated.